WordPress/src/androidTest/java/org/wordpress/android/util/EncryptionUtilsTest.java

```diff
@@ -4,58 +4,62 @@
 
 import androidx.test.ext.junit.runners.AndroidJUnit4;
 
-import org.json.JSONException;
 import org.json.JSONArray;
+import org.json.JSONException;
 import org.json.JSONObject;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.libsodium.jni.NaCl;
+import org.libsodium.jni.Sodium;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 import static junit.framework.Assert.fail;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
 @RunWith(AndroidJUnit4.class)
 public class EncryptionUtilsTest {
```

Comment:
Side note: At the moment this class sits inside `androidTest folder, which makes it part of the Instrumentation tests, I think this would fit better inside `test` as part of unit tests. 
Thoughts?

Reply:
My bad, moving the tests to the `test` folder can't be done. The library is an `AAR` file (Android ARchive) which means it needs to be run on an emulator/device. 🤦‍♀ 
I noticed when I tried to move the library and I ran the tests, which displayed the following error: 
`java.lang.UnsatisfiedLinkError: no sodiumjni in java.library.path` 

It is funny, there happen to be person who reported this, it is not a library issue though. 
https://github.com/joshjdevl/libsodium-jni/issues/95